modules/gitops-storing-and-retrieving-argo-cd-logs.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * cicd/gitops/viewing-argo-cd-logs-using-the-kibana-dashboard.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-storing-and-retrieving-argo-cd-logs_{context}"]
+= Storing and retrieving Argo CD logs
+
+You can use the Kibana dashboard to store and retrieve Argo CD logs.
+
+.Procedure
+
+. In the {product-title} web console, navigate to the {rh-app-icon} menu -> *Observability* -> *Logging*. The Kibana dashboard is opened in a new window.
+
+. Create an index pattern.
+
+.. To display all the indeces, define the index pattern as "***" . Click *Next step* to continue.
```

Comment:
Nitpick: in my opinion, there is no need to say that the purpose of clicking `*Next step*` is `to continue` because this UI label seems self-explanatory to me as a reader.
```suggestion
.. To display all the indeces, define the index pattern as "***" and click *Next step*.
```